### DISCUSSION TITLE
建议增加导入缓存功能。

### DISCUSSION THREAD

**Discussion by piaoliuchong** [Guovin/iptv-api#769] [General]
建议可通过首次导入缓存文件，直接进行二次demo的筛选，现在好像，每次运行都会载入一次直播源列表。

**Comment by Guovin** [comment]
对于访问可能出现限制情况的来源会采用缓存的形式，例如组播和酒店，同时它们更新的频率比较低。但订阅源更新频率比较高，有可能一天一更，且成功率较高，不会出现限制的情况，所以目前没有对其进行缓存。

**Comment by Guovin** [comment]
后续可能增加的是当请求某个订阅失败时，才会使用其历史缓存

**Reply by Guovin** [reply]
或者关闭网络请求时会使用缓存

**Comment by piaoliuchong** [comment]
现在是ipv4  ipv6不分开输出，只能筛选第二遍，缓存当天用的。缓存二次利用，还可以用在筛选二次突然想添加的频道列表，或者单个频道。

**Reply by Guovin** [reply]
我不太理解你说的情况，ipv4和ipv6不分开并不影响下次结果的生成，缓存和模板中的频道也并不冲突，最终还是以模板中的频道为准。

**Comment by Guovin** [comment]
与 #755 中的1重复
